api/client/stack/config.go

```diff
@@ -33,7 +33,7 @@ func newConfigCommand(dockerCli *client.DockerCli) *cobra.Command {
 }
 
 func runConfig(dockerCli *client.DockerCli, opts configOptions) error {
-	bundle, err := loadBundlefile(dockerCli.Err(), opts.namespace, opts.bundlefile)
+	bundle, err := loadBundlefile(dockerCli.Out(), opts.namespace, opts.bundlefile)
```

Comment:
For example, if someone is trying to get the output of the bundle into a file, they would direct stdout to a file, but then their will wouldn't be valid because this logging line would be included.

If it goes to stderr the file will be valid JSON.